### PR TITLE
Issue 1253 api

### PR DESCRIPTION
**Pull request by taolx0**
#1253

### CODE REVIEW

File: sqle/api/controller/v1/sync_instance.go [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+// TriggerSyncInstance trigger sync instance
+// @Summary 触发同步实例
+// @Description trigger sync instance
+// @Id triggerSyncInstanceV1
+// @Tags sync_instance
+// @Security ApiKeyAuth
+// @param sync_instance body v1.TriggerSyncInstanceTaskReqV1 true "trigger sync instance request"
+// @Success 200 {object} controller.BaseRes
+// @router /v1/task/sync_instance/trigger [post]
```
Comment (sjjian) [inline sqle/api/controller/v1/sync_instance.go:86]: /v1/sync_instances/{task的id}/trigger

File: sqle/api/controller/v1/sync_instance.go [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+// UpdateSyncInstanceTask update sync instance task
+// @Summary 更新同步实例任务
+// @Description update sync instance task
+// @Id updateSyncInstanceTaskV1
+// @Tags sync_instance
+// @Security ApiKeyAuth
+// @param sync_task body v1.UpdateSyncInstanceTaskReqV1 true "update sync instance request"
+// @Success 200 {object} controller.BaseRes
+// @router /v1/task/sync_instance [patch]
```
Comment (sjjian) [inline sqle/api/controller/v1/sync_instance.go:52]: /v1/sync_instance/{task的id}/

File: sqle/api/controller/v1/sync_instance.go [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+	Data []InstanceTaskResV1 `json:"data"`
+}
+
+type InstanceTaskResV1 struct {
+	ID                  int    `json:"id" example:"1"`
+	Source              string `json:"source" example:"actiontech-dmp"`
+	Version             string `json:"version" example:"1.23.1"`
+	URL                 string `json:"url" example:"http://10.186.62.56:10000"`
+	DbType              string `json:"db_type" example:"mysql"`
+	LastSyncStatus      string `json:"last_sync_status" enums:"success,fail" example:"success"`
```
Comment (sjjian) [inline sqle/api/controller/v1/sync_instance.go:102]: sync失败是否需要展示message

**Review by ColdWaterLW — APPROVED** [review]

**Review by sjjian — APPROVED** [review]
lgtm

**Pull request merged**